dsp_permissions_scripts/doap/doap_set.py

```diff
@@ -35,3 +38,26 @@ def apply_updated_scopes_of_doaps_on_server(doaps: list[Doap], host: str, dsp_cl
         except ApiError as err:
             logger.error(err)
     logger.info(f"Finished updating scopes of {len(doaps)} DOAPs on {host}")
+
+
+def create_new_doap_on_server(
+    target: NewDoapTarget,
+    shortcode: str,
+    scope: PermissionScope,
+    dsp_client: DspClient,
+) -> Doap | None:
+    proj_iri, _ = get_project_iri_and_onto_iris_by_shortcode(shortcode, dsp_client)
+    payload = {
+        "forGroup": target.group.val.replace("knora-admin:", KNORA_ADMIN_ONTO_NAMESPACE) if target.group else None,
```

Comment:
I already remarked in the last PR, that this replacement is a low-level implementation detail that could be hidden away nicely with a function

Reply:
Sorry for silently skipping your comment in the previous PR. I always appreciate such input. I will try it out in the above-mentioned follow-up PR :-)

Reply:
no worries! IMO these kinds of "cosmetic" suggestions one should always be free to skip - and even silently, to keep overhead minimal.